### PR TITLE
[Copy Update] Download Intel IoT G Contact Us form

### PR DESCRIPTION
**Pull request by ClementChaumel**
## Done

- Updated the copy according to the [copydoc](https://docs.google.com/document/d/12HsccVGTbjOxRX0go2m7ywFFoqrtU2JIizwIFtVWnWs/edit?disco=AAAAcF8vgUE&usp=comment_email_document&ts=62e97c6d&usp_dm=false)

## QA

- go on [/download/iot/intel-iotg](https://ubuntu-com-11928.demos.haus/download/iot/intel-iotg)
- click "contact us" 
- check that the form matches the copydoc
- try to progress to the last page of the form without filling in any details
- you should see a validation message on the "Tell us more about your project" field
- fill out the field and try to progress again
- you should be able to progress

## Issue / Card

Fixes canonical/commercial-squad#687

### CODE REVIEW

**Comment by ClementChaumel** [comment]
@sowasred2012 I added a `required` attribute on the "tell us more about your project" textarea but it doesn't seem to work. Do you know how I can specify that a field is required before pagination?

**Review by leiaru-c — COMMENTED** [review]
This looks good, it does tell you that "The tell me more about your project" field is needed to progress onto the next page. Small detail though, the style used for the "Tell us more about your project" is not the same weight as other questions. It should follow the same style choices to make it clear it's a different section from the previous question. Is it also possible to make it explicit that only the "Tell us more" field is required (thinking of an asterisk or something along those lines...)?

**Review by leiaru-c — APPROVED** [review]

File: static/js/src/dynamic-forms.js [hunk 1]
```diff
@@ -273,6 +283,49 @@
...
+              inputValid = true;
+            }
+
+            if (input.type === "radio" && input.checked) {
+              inputValid = true;
+            }
+
+            if (input.type === "text" && input.value) {
+              inputValid = true;
+            }
```
Comment (minkyngkm) [inline static/js/src/dynamic-forms.js:310]: It seems only the textarea is required in this form but is this for future use? just wondering!

**Review by minkyngkm — COMMENTED** [review]
Looking good! but the label(question) "Tell us more about your project" is not printed in the `comments_from_lead__c` 

This is the payload. 
image

Please disregard if it's not necessary! :)

File: static/js/src/dynamic-forms.js (hunk 1 above)
Comment (sowasred2012) [reply static/js/src/dynamic-forms.js:310]: ah yes - yeah, this function should make it possible for us to add required fields to other forms in future.

**Comment by sowasred2012** [comment]
> Looking good! but the label(question) "Tell us more about your project" is not printed in the `comments_from_lead__c`
> 
> This is the payload. image
> 
> Please disregard if it's not necessary! :)

Good point! I'll update this morning

**Comment by minkyngkm** [comment]
Screenshot 2022-08-09 at 11 54 35 am

Thanks @sowasred2012 I see all the the labels and values now but this two things in the red box seems to be the values of the other two questions below and these values are printed twice. Should it be fixed or do you think it's okay?

**Review by minkyngkm — APPROVED** [review]
LGTM - Thanks @sowasred2012 🏅 🙌

**Pull request merged**